cluster/log-dump/log-dump.sh

```diff
@@ -512,6 +513,9 @@ function dump_nodes_with_logexporter() {
   sed -i'' -e "s@{{.DumpSystemdJournal}}@${dump_systemd_journal}@g" "${KUBE_ROOT}/cluster/log-dump/logexporter-daemonset.yaml"
   sed -i'' -e "s@{{.ExtraLogFiles}}@${extra_log_files}@g" "${KUBE_ROOT}/cluster/log-dump/logexporter-daemonset.yaml"
   sed -i'' -e "s@{{.ExtraSystemdServices}}@${extra_systemd_services}@g" "${KUBE_ROOT}/cluster/log-dump/logexporter-daemonset.yaml"
+  if [[ -n "${compute_zone}" ]]; then
+    sed -i'' -e "s@nodeSelector:.*@nodeSelector: {topology.kubernetes.io/zone: ${compute_zone}}@" "${KUBE_ROOT}/cluster/log-dump/logexporter-daemonset.yaml"
```

Comment:
@tosi3k @jkaniuk I checked [docs](https://kubernetes.io/docs/reference/kubernetes-api/labels-annotations-taints/#failure-domainbetakubernetesiozone) and it looks like 'topology.kubernetes.io/zone' is a label that is not cloud-provider specific (this is the same flag for GCE, AWS and other cloud providers).

Please note that up to 1.16, 'failure-domain.beta.kubernetes.io/zone' is a label that should be used, so I think it makes sense to make it possible to easily override this flag so that the script can be used also for old versions without modifications.

Maybe something like:
ZONE_NODE_SELECTOR_LABEL=${ZONE_NODE_SELECTOR_LABEL:-topology.kubernetes.io/zone} and using it here.

This way, this script will be still version-agnostic which I think is a good property.

Reply:
Done.